es-design-system/pages/molecules/es-collapse.vue

```diff
@@ -8,37 +8,118 @@
                 bootstrap-vue collapse
             </b-link>
         </p>
-        <form>
-            <label for="suggestedVisibleInput">
-                <input
-                    id="suggestedVisibleInput"
-                    v-model="suggestedVisible"
-                    type="checkbox">
-                Toggle collapse programmatically (will be honored until a manual expand or collapse)
-            </label>
-        </form>
-        <EsCollapse
-            id="testId"
-            :visible="suggestedVisible"
-            class="p-450 my-450"
-            @shown="shownEvent"
-            @toggled="toggledEvent">
-            <template #title>
-                <h2 class="mb-0">
-                    My Title
-                </h2>
-            </template>
-            <!-- eslint-disable max-len -->
+        <div class="mt-450 mb-450">
+            <h2>
+                Default EsCollapse
+            </h2>
+            <p>
+                A normal EsCollapse component. Click it to toggle showing it's contents!
+            </p>
+            <EsCollapse
+                id="defaultCollapse"
+                class="ml-450 mr-450">
+                <template #title>
+                    <h2 class="mb-0">
+                        My Title
+                    </h2>
+                </template>
+                <!-- eslint-disable max-len -->
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi in aliquam ex. Nullam vestibulum ex mi, ut suscipit libero condimentum id.
+                    Pellentesque eu diam vel nisi molestie porta eget sed odio. Quisque congue risus id metus facilisis, non imperdiet libero rutrum. Mauris
+                    vitae ante porttitor, consectetur purus faucibus, euismod ex. Orci varius natoque penatibus et magnis dis parturient montes, nascetur
+                    ridiculus mus. Nulla ullamcorper elit sed viverra finibus. Mauris vitae tortor mauris. Cras suscipit nibh nec nisi cursus ornare.
+                    Maecenas quis turpis sit amet sapien dapibus sollicitudin viverra eu justo. Vivamus posuere metus sit amet purus tempus volutpat.
+                    Donec eleifend elit quam.
+                </p>
+                <!-- eslint-enable max-len -->
+            </EsCollapse>
+        </div>
+
+        <div class="mt-450 mb-450">
+            <h2>
+                Programmatic EsCollapse
```

Comment:
Nit: We could axe the "EsCollapse" here since everything on this page is EsCollapse, and it could be something like "Programmatic, with user override"